src/panels/alarm-table/action_mgr.js

```diff
@@ -31,28 +32,57 @@ export class ActionMgr {
     // We should only ack alarms that are not already acked
     let acknowledgeableRows = _.filter(this.rows, row => row.alarm.ackTime === void 0);
     this.addOptionToContextMenu('General', 'Acknowledge', acknowledgeableRows,
-        (row) => self.ctrl.acknowledgeAlarm(row.source, row.alarmId));
+        (row, callback) => {
+          self.ctrl.acknowledgeAlarm(row.source, row.alarmId).then(() => {
+            callback(null)
```

Comment:
Add `;` to complete statements for consistency.

(Here and other lines as well.)

Reply:
Updated: https://github.com/OpenNMS/opennms-helm/pull/254/commits/5eb67e89dac946f0c4def6b219c2e0f7824745cb